src/accessors/IRoomRead.ts

```diff
@@ -12,8 +12,8 @@ export interface IRoomRead {
 
     getByName(name: string): IRoom;
 
-    getMessages(roomId: string): Array<IMessage>;
+    getMessages(roomId: string, pagesize?: number): IIterator<IMessage>;
```

Comment:
@graywolf336 I do like the pagesize being a member of the iterator. This simplifies `next()`